src/NServiceKit.Text/Common/DateTimeSerializer.cs

```diff
@@ -155,7 +163,12 @@ public static DateTime ParseShortestXsdDateTime(string dateTimeStr)
 
             try
             {
-                return DateTime.Parse(dateTimeStr, null, DateTimeStyles.AssumeLocal).Prepare();
+                var parsed = DateTime.Parse(dateTimeStr, null, DateTimeStyles.AssumeLocal);
+                if (JsConfig.AssumeUtc)
+                {
+                    parsed = DateTime.SpecifyKind(parsed, DateTimeKind.Utc);
+                }
```

Comment:
This if statement was a little counter-intuitive. I expected DateTime.Parse(dateTimeStr, null, DateTimeStyles.AssumeUniversal) to be what I wanted, but that applied the TimeZone offset, which was not what I wanted.